hc-nested-list-behavior.html

```diff
@@ -69,13 +136,14 @@
      */
     _computeItems(parts, items = parts.root, idName = this.itemIdName) {
       // Can't calculate items without parts or the name of the id property.
-      if (!parts) return [];
-
+      if (!parts || !items) return [];
```

Comment:
It shouldn't be, but theoretically yes it could be, found the case while testing.